Makefile

```diff
@@ -1,7 +1,7 @@
 .PHONY: vendor
 vendor:
 	docker build -t ebpf-exporter-build .
-	docker run --rm -v $(CURDIR):/go/ebpf_exporter --workdir /go/ebpf_exporter --entrypoint /bin/bash ebpf-exporter-build -c "go get -u ./... && go mod vendor && go mod tidy"
+	docker run --rm -v $(CURDIR):/go/ebpf_exporter --workdir /go/ebpf_exporter --entrypoint /bin/bash ebpf-exporter-build -c "CGO_CFLAGS="-I /usr/include/bpf" CGO_LDFLAGS="-lbpf" go get -u ./... && go mod vendor && go mod tidy"
```

Comment:
Any non-standard env variables should be explained. If they are sprinkled in multiple places and required for the build, then it's probably best to document the reasons in the README. I was able to find the following issue for `LDFLAGS`:

* https://github.com/aquasecurity/libbpfgo/issues/1

Not really sure why `CFLAGS` is necesssary.